src/content/en/2020/css.md

```diff
@@ -405,15 +440,17 @@ Lengths are the only types of CSS values for which we can omit the unit when the
 {{ figure_markup(
   image="zero-lengths.png",
   caption="Relative popularity of 0 lengths by unit as a percent of occurrences on mobile pages.",
-  description="Pie chart showing 0 lengths with no unit (AKA unitless) used 88.7% of the time on mobile pages, the px unit at 10.7%, and other units on 0.5% of instances.",
+  description="Pie chart showing 0 lengths with no unit (AKA unitless) used 88.7% of the time on mobile pages, the `px` unit at 10.7%, and other units on 0.5% of instances.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRpe_HsNGpekn6YZV9k6QGmcZPxalqnDrL7DrDY-7X65RZEf_-aGfWuEvhk-yWV83ctIceE1bppCLpj/pubchart?oid=1935151776&format=interactive",
   sheets_gid="313150061",
   sql_file="units_zero.sql"
 ) }}
 
 ### Calculations
 
-When the [`calc()`](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()) function was introduced for performing calculations between different units in CSS, it was a revolution. Previously, only preprocessors were able to accommodate such calculations, but the results were limited to static values and unreliable, since they were missing the dynamic context that is often necessary.
+When the [`calc()`](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()) function was introduced for performing calculations between different units in CSS, it was a revelation. Previously, only preprocessors were able to accommodate such calculations, but the results were limited to static values and unreliable, since they were missing the dynamic context that is often necessary.
+
+{# TODO (authors)—revolution or revelation. I think revelation (aka "a great success") but not 100% sure (could be "A sudden, vast change in a situation, a discipline, or the way of thinking and behaving."): https://wikidiff.com/revolution/revelation #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
revolution is correct

Reply:
revolution

Reply:
OK suggesting reverting and removing TODO:

```suggestion
When the [`calc()`](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()) function was introduced for performing calculations between different units in CSS, it was a revolution. Previously, only preprocessors were able to accommodate such calculations, but the results were limited to static values and unreliable, since they were missing the dynamic context that is often necessary.
```